### DISCUSSION TITLE
Only One Page is working and the rest aren't

### DISCUSSION THREAD

**Discussion by RandomDude604** [community/community#45944] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I don't know what's happening but I cannot get the rest of my pages to work. Only the home page does, but the others lost all .CSS and .JS files and breaks switching pages.
![image](https://user-images.githubusercontent.com/70555524/215865833-48673579-22e1-4ef3-b91a-4cb7d202a8cc.png)
This Is the home page, and the other pages look like this: 
![image](https://user-images.githubusercontent.com/70555524/215865914-0bbee091-b88e-433e-bdc5-cdd4767978ef.png)
After clicking a link it then turns into this:
![image](https://user-images.githubusercontent.com/70555524/215866013-40a878b9-5367-49bb-bb85-547ccd59e772.png)
Anyone know how to fix this?

**Comment by radar-blip** [comment, marked as answer]
Its just a matter of getting your links correct. With your current setup, you'll need to use relative paths in the links to your css file.
So, in your shop.html file, 
change this: `<link rel="stylesheet" href="/main.css">`
to this: `<link rel="stylesheet" href="../main.css">`

If this works, do the same for the sidenav.css

Repeat same steps in your about.html page.

Note: This should get your css working but your html links will need relative paths too.


Also, your index.html will need to include the link to it, so one can navigate back to it from other pages:
`<a href="index.html">Home</a>`

**Reply by RandomDude604** [reply]
Thanks! This helped a lot!

**Reply by radar-blip** [reply]
No problem. Glad to help.

One more thing, the about page needs this link: `<a href="../About-Page/about.html">About</a>`

**Reply by RandomDude604** [reply]
Yup got the stuff fixed up. Again, thank you for the help!
